SpatialGDK/Source/Public/Interop/Connection/ConnectionConfig.h

```diff
@@ -21,6 +21,8 @@ struct FConnectionConfig
 		FParse::Value(CommandLine, TEXT("workerType"), WorkerType);
 		FParse::Value(CommandLine, TEXT("workerId"), WorkerId);
 		FParse::Bool(CommandLine, TEXT("useExternalIpForBridge"), UseExternalIp);
+		FParse::Bool(CommandLine, TEXT("enableProtocolLogging"), EnableProtocolLoggingAtStartup);
+		FParse::Value(CommandLine, TEXT("protocolLoggingPrefix"), ProtocolLoggingPrefix);
```

Comment:
Could you make a ticket to document this or document it with this PR?

Reply:
Filed https://improbableio.atlassian.net/browse/UNR-779 for documenting available GDK command line arguments in general.